OpenLaTeXStudio/EditorModule/src/latexstudio/editor/menu/SaveAs.java

```diff
@@ -0,0 +1,51 @@
+/* @author rbanna01
+ * See file LICENSE for copying permission.
+ */
+package latexstudio.editor.menu;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.File;
+import javax.swing.JFileChooser;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import latexstudio.editor.ApplicationLogger;
+import latexstudio.editor.EditorTopComponent;
+import latexstudio.editor.TopComponentFactory;
+import latexstudio.editor.files.FileChooserService;
+import latexstudio.editor.files.FileChooserService.DialogType;
+import latexstudio.editor.files.FileService;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionReferences;
+import org.openide.awt.ActionRegistration;
+import org.openide.util.NbBundle.Messages;
+
+@ActionID (
+        category = "File",
+        id = "latexstudio.editor.SaveAs"
+)
+@ActionRegistration (
+        iconBase = "latexstudio/editor/resources/icons/save.png",
+        displayName= "#CTL_SaveAs"
+)
+@ActionReferences ({
+        @ActionReference(path="Menu/File", position = 1400),
```

Comment:
Can we change the position to 1301? This will make it appear above the separator, that separates operation on files and generating PDF in the File menu.